tests/test_spherePoint.py

```diff
@@ -134,15 +134,36 @@ def testLonLatConstructorErrors(self):
         with self.assertRaises(pexEx.InvalidParameterError):
             SpherePoint(-42.0, -inf, degrees)
 
-    def testInit1ArgErrors(self):
-        """Test if 1-argument form of __init__ handles invalid input.
-        """
+    def testPoint3DConstructor(self):
+        # test poles
```

Comment:
Did you mean to remove the docstring?

Reply:
I meant to not include one. I think the name of the method suffices.